src/components/Input/DropDown/constants.js

```diff
@@ -0,0 +1,2 @@
+export const VARIANTS_WITH_LABEL_INSIDE = [0, 1];
+export const VARIANTS_WITH_BORDER = [0, 2];
```

Comment:
I've always felt like these integer variants were confusing... perhaps we could assign them as values of constants that possess semantic names?

Reply:
Yeah, for me they also look confusing. I just didn't want to change them because I thought there was some convention or some other reason behind that decision. If you don't mind I'll create semantic names for them

Reply:
The variants began as integers and there was a plan to change them, but it was never carried out. Yes, please create semantic names while ensuring backward compatibility 😄